operators/test/e2e/stack/pause.go

```diff
@@ -1,3 +1,7 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
```

Comment:
Thanks you to your IDE :)? That's the good license header. So we can remove the one below. Same in `operators/test/e2e/stack/volume.go`.

Reply:
Actually I turned off the Intellij copyright header support and switchted to `elastic/go-licenser` and a file watch ...